pkgs/tools/graphics/ploticus/default.nix

```diff
@@ -7,9 +7,10 @@
 , libjpeg
 , gd
 , freetype
+, runCommand
 }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: rec {
```

Comment:
You can use `finalAttrs.version` instead of relying on `rec`. That will make it update with `overrideAttrs` too.